k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapFolderTest.java

```diff
@@ -1238,4 +1240,38 @@ private void prepareImapFolderForOpen(int openMode) throws MessagingException, I
     private void assertCheckOpenErrorMessage(String folderName, MessagingException e) {
         assertEquals("Folder " + folderName + " is not open.", e.getMessage());
     }
+
+    @Test
+    public void equals_returnsFalse_whenCaseDifferent() {
```

Comment:
We usually use naming pattern for tests that would make this `equals_withDifferentCase_shouldReturnFalse`.